dub.json

```diff
@@ -13,7 +13,7 @@
     "stringImportPaths": ["resources"],
 
     "preGenerateCommands": [
-        "git describe --tags > resources/VERSION"
+        "cd $PACKAGE_DIR && git describe --tags > resources/VERSION"
```

Comment:
I though DUB would set `$PACKAGE_DIR` as an environment variable when executing the pre generate command. But it looks like it expands the variable before.